sphinx/util/i18n.py

```diff
@@ -180,10 +182,13 @@ def babel_format_date(date: datetime, format: str, locale: str,
         return formatter(date, format, locale=locale)
     except (ValueError, babel.core.UnknownLocaleError):
         # fallback to English
+        logger.warning(__('Invalid locale for babel. Fallback locale to `en`.'),
```

Comment:
Perhaps you could indicate the bad locale? because such warming is not necessarily helpful otherwise.